transforms/code/code_quality/README.md

```diff
@@ -76,7 +46,24 @@ the following command line arguments are available in addition to
 * "--tokenizer" - input a tokenizer to convert the data into tokens. The default tokenizer is `codeparrot/codeparrot`
 * "--hf_token" - input the Hugging Face auth token to download the tokenizer. This option is only required for the tokenizer's whose access is restricted in Hugging Face.
 
-## Executing S3 examples
+### Running the samples
+To run the samples, use the following `make` targets
+
+* `run-cli-sample` - runs src/code_quality_transform.py using command line args
+* `run-local-ray-sample` - runs src/code_quality_local_ray.py
+* `run-s3-ray-sample` - runs src/code_quality_s3_ray.py
+    * Requires prior invocation of `make minio-start minio-load` to load data into local minio for S3 access.
```

Comment:
No one is going to read this. I suggest removing it and start minio as part of the command

Reply:
Agreed, but this and `make help` for this target suggest minio must be started.  So I don't think they can actually run this target w/o being aware of the fact that minio has to be started (assuming they read the one liner).

Reply:
Remember. They can't read for the life of them